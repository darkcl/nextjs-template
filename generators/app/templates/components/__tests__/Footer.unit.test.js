import React from 'react';
import toJson from 'enzyme-to-json';
import { render } from 'enzyme';

import Footer from '../Footer';

describe('<Footer />', () => {
  it('should render correctly', () => {
    const footer = render(<Footer />);
    expect(toJson(footer)).toMatchSnapshot();
  });
});
