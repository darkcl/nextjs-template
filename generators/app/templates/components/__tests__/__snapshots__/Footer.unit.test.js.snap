// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render correctly 1`] = `
<div
  class="footer"
>
  <div
    class="container"
  >
    footer.text
  </div>
</div>
`;
